### PR TITLE
Correct links after release of new version of ISHDeploy module

### PR DESCRIPTION
**Pull request by vmaksymenko**

### CODE REVIEW

File: _posts/2000-01-08-DITA.md [hunk 1]
```diff
@@ -13,17 +13,10 @@ fa-icon: file-code-o
 
 ISHDeploy is a PowerShell deployment automation module for SDL Knowledge Center Content Manager (LiveContent Architect, Trisoft InfoShare).
```
Comment (ddemeyer) [inline _posts/2000-01-08-DITA.md]: SDL Tridion Docs Content Manager (Knowledge Center Content Manager, LiveContent Architect, Trisoft InfoShare)

File: _posts/2000-01-08-DITA.md [hunk 2]
```diff
@@ -13,17 +13,10 @@ fa-icon: file-code-o
...
-  - [Getting started](https://sdl.github.io/ISHDeploy/12.0/)
-- SDL Knowledge Center 13 Content Manager 13.0.*
-  - [Install ISHDeploy.13.0](https://www.powershellgallery.com/packages/ISHDeploy.13.0/)
-  - [Getting started](https://sdl.github.io/ISHDeploy/13.0/)
-
-Go to [ISHDeploy GitHub](https://github.com/sdl/ISHDeploy/)
-
-Or simply get started right away: `git clone https://github.com/sdl/ISHDeploy.git`
-
+- SDL Knowledge Center Content Manager
```
Comment (ddemeyer) [inline _posts/2000-01-08-DITA.md]: SDL Tridion Docs Content Manager

File: _posts/2000-01-08-DITA.md [hunk 3]
```diff
@@ -13,17 +13,10 @@ fa-icon: file-code-o
...
+  - [Install ISHDeploy](https://www.powershellgallery.com/packages/ISHDeploy/)
+  - [Getting started](https://sdl.github.io/ISHDeploy/)
+  
 ## ISHRemote
 
 ISHRemote is a business automation module on top of SDL Knowledge Center Content Manager (LiveContent Architect, Trisoft InfoShare).
```
Comment (ddemeyer) [inline _posts/2000-01-08-DITA.md]: SDL Tridion Docs Content Manager (Knowledge Center Content Manager, LiveContent Architect, Trisoft InfoShare)

**Review by ddemeyer — CHANGES_REQUESTED** [review]
Can you do the rebranding, this makes sure the module is found in search/find operations. Thanks

File: _posts/2000-01-08-DITA.md (hunk 1 above)
Comment (vmaksymenko) [reply _posts/2000-01-08-DITA.md]: Done

File: _posts/2000-01-08-DITA.md (hunk 2 above)
Comment (vmaksymenko) [reply _posts/2000-01-08-DITA.md]: Done

File: _posts/2000-01-08-DITA.md (hunk 3 above)
Comment (vmaksymenko) [reply _posts/2000-01-08-DITA.md]: Done

**Pull request merged**